.github/workflows/node.js.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [14.x, 16.x]
```

Comment:
Do you want this set to `14` and `16`?

Reply:
I've set it back to just 14, the build was initially failing and I thought it was maybe to do with the different node versions (it was actually just that I had forgotten to reinstall the package-lock.json file).